galata/test/documentation/utils.ts

```diff
@@ -56,3 +58,19 @@ export function positionMouse(position: { x: number; y: number }): string {
       stroke="#ffffff" stroke-width=".54745" style="paint-order:markers fill stroke" />
 </svg>`;
 }
+
+export async function setLeftSidebarWidth(
+  page: Page,
+  width = 251
+): Promise<void> {
+  const splitHandle = await page.$('.lm-SplitPanel-handle');
+  const handleBBox = await splitHandle.boundingBox();
+
+  await page.mouse.move(
+    handleBBox.x + 0.5 * handleBBox.x,
```

Comment:
Looks like we might need the `width` here? (not tested though)

```suggestion
    handleBBox.x + 0.5 * handleBBox.width,
```